src/Humanizer.Tests/ApiApprover/PublicApiApprovalTest.approve_public_api.approved.txt

```diff
@@ -243,6 +243,23 @@ public class In
     public System.DateTime TheYear(int year) { }
 }
 
+public class Vocabularies
+{
+    public Vocabularies() { }
```

Comment:
Can you please remove this default constructor from public surface?
This class isn't intended to have any instance, right?